docs/resources/flexible_ip.md

```diff
@@ -0,0 +1,77 @@
+---
+page_title: "Scaleway: scaleway_flexible_ip"
+description: |-
+  Manages Scaleway Flexible IPs.
+---
+
+# scaleway_flexible_ip
+
+Creates and manages Scaleway flexible IPs.
+For more information, see [the documentation](https://developers.scaleway.com/en/products/flexible-ip/api).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_flexible_ip" "main" {
+    reverse = "my-reverse.com"
+}
+```
+
+### With baremetal server
+
+```hcl
+data "scaleway_account_ssh_key" "main" {
+    name = "main"
+}
+
+data "scaleway_baremetal_os" "by_id" {
+    zone = "fr-par-2"
+    os_id = "03b7f4ba-a6a1-4305-984e-b54fafbf1681"
```

Comment:
Could you use an explicit name for os? Not an uuid but something like ubuntu for instance.